aten/src/ATen/native/README.md

```diff
@@ -58,6 +58,13 @@ signature.
   don't commit them to memory.
 - `Tensor[]`.  A `Tensor[]` argument translates into a C++ argument of type `ArrayRef<Tensor>`
   (a.k.a. `TensorList`)
+- `Tensor` or `Tensor?` must sometimes be annotated to indicate aliasing and mutability.
+  In general annotations can be defined via the following four situtations
+  `Tensor(a)` - Tensor is in set a
```

Comment:
I know this is coming from the original JIT docs (so not your issue), but it could use some improvement:
- "is in set" is never defined.
- don't use 'a', it's a word and makes the statement harder to parse.